app/services/token.rb

```diff
@@ -27,28 +27,15 @@ def self.generate_token_pdf(person, media)
            'A4' =>  { :width => 90, :height =>29, :leftMargin => 2, :rightMargin => 3 , :spacing => 3 } }
```

Comment:
we might actually want to move this somewhere else, too. shouldn't the printing layout specify which paper to use? but that's fine for the moment :)